src/Tickets/Commerce/Flag_Actions/Flag_Action_Abstract.php

```diff
@@ -65,7 +97,7 @@ public function get_post_types() {
 	 * {@inheritDoc}
 	 */
 	public function should_trigger( Status_Interface $new_status, $old_status, $post ) {
-		if ( ! $this->has_flags( $new_status ) ) {
+		if ( ! $this->has_flags( $new_status, $post ) ) {
```

Comment:
not sure if you've addressed this yet, but it looks like this function call doesn't match the signature.

Reply:
I think it got fixed in another PR later.